tests/core/test_nlg.py

```diff
@@ -115,8 +115,25 @@ def test_nlg_fill_template_text(slot_name, slot_value):
     assert result == {"text": str(slot_value)}
 
 
+@pytest.mark.parametrize(
+    "template_text, expected",
+    [
+        ('{{"variable":"{slot_1}"}}', '{"variable":"foo"}'),
```

Comment:
Shouldn't the result be `{{"variable":"foo"}}`?

Reply:
I know this look confusing but the test result should be correct. Basically in original value (template_text), you are escaping `{` with `{{` and expected to result in `{`. The 3rd one also shows the similar result.

Reply:
and change log has been updated as well as I touched the other case and changed from `_w_` to `_with_` to be more clear and consistent.

Reply:
Thanks, got it now :-)

Reply:
what happens if we have `{{slot_1}}`?